src/ECharts/Commands.purs

```diff
@@ -430,33 +436,69 @@ textMiddle = set "textBaseline" $ toForeign "middle"
 brush ∷ ∀ i. DSL TP.BrushI → DSL (brush ∷ I|i)
 brush = set "brush" <<< buildObj
 
+brushType ∷ ∀ i. DSL TP.BrushToolboxI → DSL (brushType ∷ I|i)
+brushType a = set "type" $ buildArr a

```

Comment:
Shouldn't `"type"` be `"brushType"`? and same beloved for some other cases

Reply:
That's by design, too much `type` fields in js. But they all are different: e.g. type for brush, type for series, type for events etc. So, in built options this is just `"type": ["blahblah"]` but that `blahblah` may be used only in brush context.